src/app/pathway-to-success/crafting-cv/page.tsx

```
@@ -0,0 +1,76 @@
+  return (
+    <div>
+      {/* uncomment this to update the contents */}
+      {/* <TextEditor initialValue={value} onEditorChange={setValue} /> */}
+
+      <div className="p-5">
+        <section>
+          <div className="text-2xl font-semibold">
+            {impressiveCVContents.title}
+          </div>
+          <div
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)